ssl/ssl_lib.c

```diff
@@ -5947,3 +5947,17 @@ int SSL_CTX_set0_tmp_dh_pkey(SSL_CTX *ctx, EVP_PKEY *dhpkey)
     ctx->cert->dh_tmp = dhpkey;
     return 1;
 }
+
+uint16_t SSL_get_grease_value(SSL *s,
+                              SSL_GREASE_INDEX index) {
+    RAND_bytes(s->grease_seed, sizeof(s->grease_seed));
+
+    // This generates a random value of the form 0xωaωa, for all 0 ≤ ω < 16.
+    uint16_t ret = s->grease_seed[index];
```

Comment:
Declaration before code, older compilers don't accept this later addition to C.

Reply:
No prob